### DISCUSSION TITLE
ettercap issue

### DISCUSSION THREAD

**Discussion by bellakamsar** [community/community#132268] [Programming Help]
### Select Topic Area

Question

### Body

Apres l'execution de ETTERCAP GRAPHICAL  je recoi ce message erreur:

 #sudo ettercap -G -w packets.pcap

ettercap 0.8.3.1 copyright 2001-2020 Ettercap Development Team

Ooops ! This shouldn't happen...
Segmentation Fault...

Please recompile in debug mode, reproduce the bug and send a bugreport

 Have a nice day!

**Comment by bellakamsar** [comment]
quelqu'un peux me supporter pour résoudre ce problème please ?

**Comment by LiteBrite82** [comment]
Hi @bellakamsar , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
